e2etests/tests/unnumbered.go

```diff
@@ -0,0 +1,286 @@
+		nodeWithP2PConnection = nodes[0]
+
+	})
+
+	ginkgo.AfterEach(func() {
+		if ginkgo.CurrentSpecReport().Failed() {
+			reporter := dump.NewK8sReporter(k8s.FRRK8sNamespace)
+			testName := ginkgo.CurrentSpecReport().FullText()
+			dump.K8sInfo(testName, reporter)
+			dump.BGPInfo(testName, []*frrcontainer.FRR{externalP2PContainer}, cs)
+		}
+	})
+
+	ginkgo.DescribeTable("BGP session is established and routes are verified", func(rc frrconfig.RouterConfigUnnumbered) {
+		iface := rc.Interface
+		remoteASN := rc.ASNLocal
+		prefixSend := []string{"5.5.5.0/24", "5555::/64"}
+
+		externalP2PContainer, err := frrcontainer.CreateP2PPeerFor(nodeWithP2PConnection.Name, iface, FRRImage)
+		Expect(err).NotTo(HaveOccurred())
+		ginkgo.By(fmt.Sprintf("create p2p %s:%s -- %s:%s)", nodeWithP2PConnection.Name, iface, iface, externalP2PContainer.Name))
+		ginkgo.DeferCleanup(func() {
+			err := frrcontainer.Delete([]*frrcontainer.FRR{externalP2PContainer})
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		ginkgo.By(fmt.Sprintf("load frrconfig to %s", externalP2PContainer.Name))
+		c, err := rc.Config()
+		Expect(err).NotTo(HaveOccurred())
+		err = externalP2PContainer.UpdateBGPConfigFile(c)
+		Expect(err).NotTo(HaveOccurred())
+
+		ginkgo.By(fmt.Sprintf("apply frrconfiguration %s", "for-"+externalP2PContainer.Name))
+		cr := frrk8sv1beta1.FRRConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "for-" + externalP2PContainer.Name,
+				Namespace: k8s.FRRK8sNamespace,
+			},
+			Spec: frrk8sv1beta1.FRRConfigurationSpec{
+				BGP: frrk8sv1beta1.BGPConfig{
+					BFDProfiles: []frrk8sv1beta1.BFDProfile{{Name: "simple"}},
+					Routers: []frrk8sv1beta1.Router{
+						{
+							ASN: infra.FRRK8sASN,
+							Neighbors: []frrk8sv1beta1.Neighbor{
+								{
+									ASN:        remoteASN,
+									Interface:  iface,
+									BFDProfile: "simple",
+									ToReceive: frrk8sv1beta1.Receive{
+										Allowed: frrk8sv1beta1.AllowedInPrefixes{
+											Mode: frrk8sv1beta1.AllowAll,
+										},
+									},
+									ToAdvertise: frrk8sv1beta1.Advertise{
+										Allowed: frrk8sv1beta1.AllowedOutPrefixes{
+											Mode: frrk8sv1beta1.AllowAll,
+										},
+									},
+								},
+							},
+							Prefixes: prefixSend,
+						},
+					},
+				},
+				NodeSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"kubernetes.io/hostname": nodeWithP2PConnection.GetLabels()["kubernetes.io/hostname"],
+					},
+				},
+			},
+		}
+		err = updater.Update([]corev1.Secret{}, cr)
+		Expect(err).NotTo(HaveOccurred())
+		ginkgo.DeferCleanup(func() {
+			err := updater.DeleteFRRConfiguration(cr)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		nodeLLA, err := container.GetIPv6Link(nodeWithP2PConnection.Name, iface)
+		Expect(err).NotTo(HaveOccurred())
+		peerLLA, err := container.GetIPv6Link(externalP2PContainer.Name, iface)
+		Expect(err).NotTo(HaveOccurred())
+		ginkgo.By(fmt.Sprintf("%s/%s/%s --- %s/%s/%s",
```

Comment:
By("validating the node and p2p container peered")

Reply:
done